tests/basic/test_bergamot_translator_app_intgemm_8bit.cpu-threads.0.sh

```diff
@@ -1,5 +1,12 @@
 #!/bin/bash
 
+#####################################################################
+# SUMMARY: Run tests for bergamot-translator-app
+# AUTHOR: jerinphilip 
+# TAGS: wasm, native, vanilla
```

Comment:
@jerinphilip Do the `TAGS` keyword mean something when it comes to running the scripts?

Reply:
Yes. TAGS are used to restrict regression tests to say cpu only or gpu only etc while running on CI. So I can restrict certain tests to 'wasm' using tags. 

The latest source allows for this. Examples are in this comment:
https://github.com/browsermt/bergamot-translator-tests/issues/6#issuecomment-788039354